test/functional/assertion-helper.js

```diff
@@ -59,10 +60,18 @@ function checkScreenshotFileCropped (filePath) {
             const height = png.height;
 
             // NOTE: sometimes an appearing dialog can cover an edge of the browser. Try to check all edges
-            return hasPixel(png, RED_PIXEL, 0, 0) && hasPixel(png, RED_PIXEL, 49, 49) && hasPixel(png, GREEN_PIXEL, 50, 50) ||
-                   hasPixel(png, RED_PIXEL, width - 1, height - 1) && hasPixel(png, RED_PIXEL, width - 50, height - 50) && hasPixel(png, GREEN_PIXEL, width - 51, height - 51) ||
-                   hasPixel(png, RED_PIXEL, width - 1, 0) && hasPixel(png, RED_PIXEL, width - 50, 49) && hasPixel(png, GREEN_PIXEL, width - 51, 50) ||
-                   hasPixel(png, RED_PIXEL, 0, height - 1) && hasPixel(png, RED_PIXEL, 49, height - 50) && hasPixel(png, GREEN_PIXEL, 50, height - 51);
+            // NOTE: tests do not work on retina displays since test code is not aware of this
+            // NOTE: tests check for absolute rgb values therefore the display needs to be set to an sRGB color profile (macOS)
+            return (
+                // topLeft
+                hasPixel(png, RED_PIXEL, 0, 0) && hasPixel(png, RED_PIXEL, 49, 49) && hasPixel(png, GREEN_PIXEL, 50, 50) ||
+                // topRight
+                hasPixel(png, RED_PIXEL, width - 1, 0) && hasPixel(png, RED_PIXEL, width - 50, 49) && hasPixel(png, GREEN_PIXEL, width - 51, 50) ||
+                // bottomRightRounded
+                hasPixel(png, RED_PIXEL, width - 1 - MARK_RIGHT_MARGIN, height - 1) && hasPixel(png, RED_PIXEL, width - 1, height - 1 - MARK_RIGHT_MARGIN) && hasPixel(png, RED_PIXEL, width - 50, height - 50) && hasPixel(png, GREEN_PIXEL, width - 51, height - 51) ||
```

Comment:
this is where I changed the checks, not to look at the corner, but at the edges near the corner